cmd/cosign/cli/fulcio/fulcio.go

```diff
@@ -92,7 +92,13 @@ func GetCert(_ context.Context, sv signature.SignerVerifier, idToken, flow, oidc
 	case flowDevice:
 		c.flow = oauthflow.NewDeviceFlowTokenGetterForIssuer(oidcIssuer)
 	case flowNormal:
-		c.flow = oauthflow.DefaultIDTokenGetter
+		if strings.Contains(oidcIssuer, "keycloak") {
```

Comment:
Why this conditional? I think we always want to show branded Red Hat response, no matter what OIDC provider is used.